templates/tracker/standard_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load bootstrap4 %}
+{% block content %}
+<form action="" method="post">
+    {% csrf_token %}
+   
+    {% bootstrap_form form %} 
+<button type="submit" class="btn btn-primary">
+    Save
+</button>
+
```

Comment:
There should probably be a closing `<form>` tag here
```suggestion
</form>
```